includes/admin/class-wp-job-manager-admin-notices.php

```diff
@@ -317,16 +332,40 @@ private static function generate_notice_from_updates( $updates ) {
 	 * Renders a notice.
 	 *
 	 * @param array $notice See `generate_notice_from_updates` for format.
+	 * @param string $notice_id  Unique identifier for the notice.
 	 * @return void
 	 */
-	private static function render_notice( $notice ) {
-		// TODO Handle different levels of notices.
-		echo '<div class="notice wpjm-admin-notice wpjm-admin-notice--info">';
+	private static function render_notice( $notice_id, $notice ) {
+		if ( empty( $notice['actions'] ) || ! is_array( $notice['actions'] ) ) {
+			$notice['actions'] = [];
+		}
+
+		$notice_class = '';
+		if ( ! empty( $notice['style'] ) ) {
+			$notice_class = 'wpjm-notice-' . $notice['style'];
```

Comment:
I think it would be nice to check if `style` is info, error, warning, success and fall back to info.

Reply:
I've committed a change for this at https://github.com/Automattic/WP-Job-Manager/pull/2401/commits/4c798a9f95176ca5b17fa66c93fa1d2ab6157a44.

Reply:
@aaronfc I've made another change to hopefully improve handling of this, so that we don't risk nonsense CSS classes. https://github.com/Automattic/WP-Job-Manager/pull/2401/commits/03898a95f2a118ce6de45554cbc7950238abc153